### DISCUSSION TITLE
Is there a possibility to change the CSS of the tab-title of a tab that is in focus?

### DISCUSSION THREAD

**Discussion by ralf3u** [geany/geany#3784] [Q&A] [answered]
Last year in May I asked this question:

Is there a possibility to change the CSS of the tab-title of a tab that is active? It is the discussion #3482.

Today I recognized that if the sidebar is visible, then two tabs have the same CSS, because two tabs are active. So, I'm looking for a selector name where I can define the CSS for the tab that is in focus. It is not `notebook tab:focus label`.

Thank you very much for an answer in advance.

**Comment by ralf3u** [comment]
This is what I already have:
```
/* design of tabs */
tab {
	padding-top:25px;
	padding-bottom:10px;
	padding-left: 35px;
}

/* design of active tabs (e.g. tab of the editor and tab of the sidebar) */
tab:checked {
	font-weight: bold;
	border-bottom: 5px solid black;
}
```

**Comment by elextr** [comment]
Please note that this is a Geany forum, it isn't a GTK CSS consulting forum, you may get a better answer elsewhere.  

Its not focus, the editor is what is focussed, try all the pseudo classes listed in [the list below this which is the nearest link target to it](https://docs.gtk.org/gtk3/css-overview.html#theme-inconsistent-checkbuttons).

**Comment by ralf3u** [comment, marked as answer]
Thanks to the plugin Xi/Tweaks it is possible to change the CSS of the tab-title of a tab that is in focus. It works for all panels, so editor, sidebar and message window. For more information visit https://github.com/xiota/geany-preview/issues/5#issuecomment-2016831500.

To change only in the sidebar the CSS of the tab-title of a tab that has focus, visit https://github.com/xiota/geany-preview/issues/5#issuecomment-2016850315.
